src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -249,6 +263,41 @@ void run_WithSyncAsArgumentWithAllArg_ShouldExecuteAllSyncers() {
     }
   }
 
+  @Test
+  void
+      run_WithCustomerSyncAsArgument_ShouldSyncCustomersAndStoreLastSyncTimestampsAsCustomObject() {
+    // preparation
+    try (final SphereClient targetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
+      try (final SphereClient sourceClient = createClient(CTP_SOURCE_CLIENT_CONFIG)) {
+
+        final SyncerFactory syncerFactory =
+            SyncerFactory.of(() -> sourceClient, () -> targetClient, Clock.systemDefaultZone());
+
+        // test
+        CliRunner.of().run(new String[] {"-s", "customers"}, syncerFactory);
+      }
+    }
+
+    // create clients again (for assertions and cleanup), since the run method closes the clients
+    // after execution
+    // is done.
```

Comment:
make it one line

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/186/commits/725bb8840fb70e50e5bff764f29ecfb0d6f27fc0